tcms/testruns/forms.py

```diff
@@ -72,6 +72,13 @@ class XMLRPCUpdateRunForm(XMLRPCNewRunForm):
         queryset=Build.objects.all(),
         required=False
     )
+    stop_date = forms.DateTimeField(
+        required=False,
+        input_formats=['%Y-%m-%d'],
+        error_messages={
+            'invalid': 'The stop date is invalid. The valid format is YYYY-MM-DD.'
```

Comment:
Use `_` (aka `ugettext_lazy`) to translate this error message. You also need to add a negative case where the stop_date format doesn't match the expected to assert that this validation works.

Reply:
Done